core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -828,13 +840,34 @@ angular.module('oppia').directive('conversationSkin', [
                 return;
               }
             }
+
+            if (!$scope.isInPreviewMode &&
+              ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE) {
+              initLearnerAnswerInfoService(
+                $scope.explorationId, ExplorationEngineService.getState(),
+                answer, interactionRulesService);
+            }
+
             NumberAttemptsService.submitAttempt();
 
             $scope.answerIsBeingProcessed = true;
             hasInteractedAtLeastOnce = true;
 
             PlayerTranscriptService.addNewInput(answer, false);
 
+            if ($scope.canAskLearnerForAnswerInfo()) {
+              $timeout(function() {
+                PlayerTranscriptService.addNewResponse(
+                  '<p translate="I18N_SOLICIT_ANSWER_DETAILS_QUESTION"></p>');
+                $scope.answerIsBeingProcessed = false;
+                $scope.$broadcast('helpCardAvailable', {
+                  helpCardHtml: '<p>Hey how did you landed on this answer?</p>',
```

Comment:
Ditto -- no HTML in JS. Also, this should be something that gets translated since it's learner facing.

Reply:
Done